setup.sh

```diff
@@ -0,0 +1,6 @@
+sudo curl -sSf https://sh.rustup.rs | sh
```

Comment:
I think it would be nice if this script checked for the presence of `cargo`, `solc`, and `wabt` before running these install commands. Then it could be no-op most of the time and this could save some CPU cycles in the CI.

`near-cli` is also not included in this setup script, so you maybe want to add that as well.

Reply:
> I think it would be nice if this script checked for the presence of cargo, solc, and wabt before running these install commands.

I think ``apt install`` do it for us, isn't it ?

Reply:
It will, yes. But I think it would be more efficient to check the output of something like `which` or to call `--version` on the program. `apt` just seems like a heavy thing to run every time in the CI.